libraries/aws/provisioning/src/aws_iot_provisioning_api.c

```diff
@@ -742,7 +742,9 @@ AwsIotProvisioningError_t AwsIotProvisioning_CreateCertificateFromCsr( IotMqttCo
 
     if( status == AWS_IOT_PROVISIONING_SUCCESS )
     {
-        IotLogDebug( "Subscribed to response topics: Operation={\"%s\"}",
+        IotLogDebug( "Subscribed to response topics: TopicFilter={%.*s}, Operation={\"%s\"}",
+                     PROVISIONING_CREATE_CERT_FROM_CSR_RESPONSE_TOPIC_FILTER_LENGTH,
+                     PROVISIONING_CREATE_CERT_FROM_CSR_RESPONSE_TOPIC_FILTER,
```

Comment:
Is this defined to something NULL-terminated?  If so, it's not necessary to provide the length.

Reply:
If you're referring to the operation parameter, it is not a user-input data but a statically allocated string data by the library (specified here: https://github.com/aggarw13/aws-iot-device-sdk-embedded-C/blob/csr-feat/demo/libraries/aws/provisioning/src/private/aws_iot_provisioning_internal.h#L82-L85)